protos/shell/shell.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package mavsdk.rpc.shell;
+
+option java_package = "io.mavsdk.shell";
+option java_outer_classname = "ShellProto";
+
+/**
+ * Shell Service allow to communicate with vehicle's system shell.
+ */
+service ShellService {
+    // Communicate with a vehicle's Shell.
+    rpc SetShellMessage(SetShellMessageRequest) returns(SetShellMessageResponse) {}
+}
+
+message SetShellMessageRequest {
+    ShellMessage shell_message = 1;
+}
+message SetShellMessageResponse {
+    ShellResult shell_result = 1;
+}
+
+message ShellMessage {
+    bool need_response = 1; // Set if the sender wants the receiver to send a response.
+    uint32 timeout = 2; // Timeout (ms).
+    string data = 3; // Serial data (70 symbols max).
+}
+
+message ShellResult {
+    // Possible results returned for shell requests
+    enum Result {
+        UNKNOWN = 0; // Unknown error
+        SUCCESS = 1; // Request succeeded
+        NO_SYSTEM = 2; // No system is connected
+        CONNECTION_ERROR = 3; // Connection error
+        DATA_TOO_LONG = 4; // Request Data too long
+        NO_RESPONSE = 5; // Response does not received
+    }
+
+    Result result = 1; // Result enum value
+    string result_str = 2; // Human-readable English string describing the result
+    ShellMessage response_shell_message = 3; // Response message (if available)
```

Comment:
Does it make sense to re-use the `ShellMessage` here? To me it looks like it should be a different type without `need_response` and `timeout`.

Reply:
Oh, I forgot to check actual usage cases in PX4/Firmware, again :)
Seems like `need_response` and `timeout` does not used it it.
I will check that.